sos/report/__init__.py

```diff
@@ -680,6 +681,21 @@ def _get_network_namespaces(self):
                 out_ns.append(line.partition(' ')[0])
         return out_ns
 
+    def _get_devices_by_fstype(self):
+        _dev_fstypes = {}
+        _devs = sos_get_command_output("lsblk -nrpo FSTYPE,NAME")['output']
```

Comment:
We usually test `sos_get_command_output` to `status` (return code of the command), like 9 lines above - but when return code is nonzero, the output shall be empty - what is handled properly further. ACK.

Reply:
check for status added